src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,44 @@
+package racingcar;
+
+public class Car {
+
+    private static final int MOVABLE_MINIMUM_RPM = 3;
+    private static final int MIN_NAME_LENGTH = 1;
+    private static final int MAX_NAME_LENGTH = 5;
+    private final String name;
+    private int position = 0;
+
+    public Car(String name) {
+        isValidCarName(name);
+        this.name = name;
+    }
+
+    public void tryToMove(int currentRPM) {
+        if (movementCheck(currentRPM)) {
```

Comment:
```suggestion
        if (isMovable(rpm)) {
             position++;
        }
```
이렇게 하면 더 깔끔해질것 같은데 어떻게 생각하시나요?